src/cljs/nr/utils.cljs

```diff
@@ -307,9 +302,19 @@
 (defn render-message
   "Render icons, cards and special codes in a message"
   [input]
-  (if (string? input)
-    (render-specials (render-icons (render-cards input)))
-    input))
+  (render-specials (render-icons (render-cards input))))
+
+(defn highlight-patterns-impl [corp runner]
+  (letfn [(regex-of [player-name] (re-pattern (str "(?i)" (regex-escape player-name))))]
+    (->> {corp [:span.corp-username corp]
+          runner [:span.runner-username runner]}
+         (filter (fn [[k _]] (not-empty k)))
```

Comment:
filter is important here if either of the names becomes `""` or nil this will end up wrapping each character with a span